.devcontainer/Dockerfile

```diff
@@ -27,5 +27,6 @@ RUN sudo apt-get update \
   && sudo apt-get install --yes pkg-config \
   && rm -rf /var/lib/apt/lists/*
 
-# Setup ZSH for local development inside the image:
+# Setup Bash/Zsh for local development inside the image:
 COPY ./scripts/ /home/vscode/
+RUN cat /home/vscode/.bashrc_devcontainer >> /home/vscode/.bashrc
```

Comment:
my understanding is that it is customary to just leave the `~/.bashrc_devcontainer` file there, and users can always `source` it from their `.bashrc`, which is actually copied over by VS Code from users' dot files repo, instead of us overwriting it here.